scripts/lib/basic_youtube_upload.py

```diff
@@ -130,20 +151,84 @@ def resumable_upload(request):
             error = 'A retriable error occurred: %s' % e
   
         if error is not None:
-            print(error)
+            debug(error)
             retry += 1
             if retry > MAX_RETRIES:
                 raise ValueError(error)
     
             max_sleep = 2 ** retry
             sleep_seconds = random.random() * max_sleep
-            print('Sleeping %f seconds and then retrying...' % sleep_seconds)
+            debug('Sleeping %f seconds and then retrying...' % sleep_seconds)
             time.sleep(sleep_seconds)
 
+def get_known_playlist(title):
+    return PLAYLIST_IDS[title]
```

Comment:
This will raise a `KeyError` if `title` doesn’t exist in `PLAYLIST_IDS`. Based on how you’re using this function, you probably want to use [`dict.get()`](https://docs.python.org/3.6/library/stdtypes.html#dict.get):

```suggestion
    return PLAYLIST_IDS.get(title)
```